Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -2495,6 +2545,8 @@ static NSStringCompareOptions compareOptionsForFindOptions(WebCore::FindOptions
     dictionaryPopupInfo.origin = rectInRootView.location();
     dictionaryPopupInfo.platformData.attributedString = WebCore::AttributedString::fromNSAttributedString(string.get());
 
+    // FIXME: Consider merging with textIndicatorForSelection in order to get
+    // a PDF snapshot instead of an attributed string replica.
```

Comment:
Is a PDF snapshot more desirable?

Reply:
Certainly will merge the two soon™️

Reply:
Yes, because it looks exactly like the PDF! No slight jump/alignment issues. We learned this lesson many years ago with web content.

The one thing that's still better about web content is that we can do a selection-only paint so you don't get irrelevant rendering in your snapshot. But this is still progress.